app/services/recommendation_service.rb

```diff
@@ -16,6 +16,7 @@ def self.recommendation(user_id)
     baselines = merge_distances_and_ratings(cocktail_ratings, df, user_id)
     return nil if baselines.empty
     weighted_ratings(baselines, user_id)
+    weighted_rating = baselines[user_id] * baselines['value']
```

Comment:
Wait, @markcyen isn't this (l.19) doing the exact same thing as line 58 inside the weighted ratings method? We should probably remove line 18 if we want to do it this way, right?

Reply:
Ah yes, I must have uncommented it and meant to delete this line. Thanks for catching. Will push up

Reply:
Thanks! Looks good now